validations/src/validations.rs

```diff
@@ -161,24 +200,21 @@ pub fn validate_dr_transaction(tx: &TransactionBody) -> Result<(), failure::Erro
 
 // Add 1 in the number assigned to a DataRequestOutput
 pub fn update_count<S: ::std::hash::BuildHasher>(
```

Comment:
_Suggestion_: a better name would be `inc_witnesses` (inc for increment/increase) or something that better describes the actual behavior

Reply:
I'm totally in for forceful expressive function names like `increment_witnesses_counter`.